src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="form"
+      action="#"
+      method="post"
+      data-qa="big"
+      name="bigForm"
+    >
+      <lebel class="searchBar">
+        <img
+          src="images/Search.svg"
+          alt="Search image"
+          class="searchBar__img searchBar__img--big"
+        >
```

Comment:
Need to fix it
<img width="533" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/fca0a2c4-5763-4f3b-9241-3fa200f3c7c9">
```suggestion
```